src/files/XmlFile.spec.ts

```diff
@@ -58,13 +58,13 @@ describe('XmlFile', () => {
             program.plugins.add({
                 name: 'allows modifying the parsed XML model',
                 afterFileParse: () => {
-                    let child = file.parser.ast.component.children.children[0];
+                    let child = file.parser.ast.component.children.childNodes[0];
                     expect(child.attributes).to.have.lengthOf(4);
                     child.setAttribute('text', undefined);
-                    expect(child.getAttribute('id').value.text).to.equal('one');
+                    expect(child.getAttribute('id').tokens.value.text).to.equal('one');
```

Comment:
`child.getAttribute('id').value` maybe? so you also test the getter.

Reply:
Will do.